ui/src/Grid/Message.elm

```diff
@@ -0,0 +1,7 @@
+module Grid.Message exposing (..)
+
+import Grid.Player exposing (..)
+
+type Msg = PlayerMoved PlayerId Position Orientation
+         | PlayerDied PlayerId
+         | NoOp
```

Comment:
/o\ d'oh, I should've realised you can't reuse modules between the two. Never mind my comments, then.